ddtrace/tracer.py

```diff
@@ -90,8 +91,16 @@ def configure(self, enabled=None, hostname=None, port=None, sampler=None,
         if enabled is not None:
             self.enabled = enabled
 
-        if hostname is not None or port is not None:
-            self.writer = AgentWriter(hostname or self.DEFAULT_HOSTNAME, port or self.DEFAULT_PORT)
+        processing_pipeline = None
+        if settings is not None and PP_KEY in settings:
+                processing_pipeline = settings[PP_KEY]
+
+        if hostname is not None or port is not None or processing_pipeline is not None:
+            self.writer = AgentWriter(
```

Comment:
I think we have some indentation issues here. Don't know why `flake8` didn't catch that!